tools/analyzer_plugin/README.md

```diff
@@ -0,0 +1,102 @@
+# OverReact Analyzer Plugin
+
+> A [Dart analyzer plugin][analyzer_plugin] for OverReact
+>
+> - [Notes/planning document](https://docs.google.com/document/d/1xHjC66eUvX_SSBRXw-sVOxRzvenWfOpu8m1BQI6GmqQ/edit#)
+> - Slack channel: [#react-analyzer-plugin](https://slack.com/app_redirect?channel=react-analyzer-plugin)
+
+
+## Try it in your package!
+1. Add over_react to your pubspec.yaml
+
+    __If you wish to develop the plugin locally, see [this section](#local-development).__
+    
+1. Enable the plugin in your analysis_options.yaml:
+    ```yaml
+    analyzer:
+      plugins:
+        over_react
+    ```
+1. Restart the Dart Analysis Server in your IDE
+   
+## Repo Structure
+
+See the [analyzer_plugin package structure documentation][analyzer_plugin_package_structure] for terminology and more info.
+
+- _over_react_: the "host" package
+- _over_react_analyzer_plugin_ (`over_react/tools/analyzer_plugin`) - the "boostrap" and "plugin" packages, merged
+        
+    We decided to merge these packages since it allows us to avoid creating a separate package for the plugin, which would have resulted in a more painful dev/release experience. (Since the plugin depends on over_react, we'd want to use monorepo to manage the packages, but we can't do that currently due to internal tooling restrictions.)
```

Comment:
```suggestion
    We decided to merge these packages since it allows us to avoid creating a separate package for the plugin, which would have resulted in a more painful dev/release experience. (Since the plugin depends on over_react, we'd want to use monorepo to manage the packages. However, we can't do that currently, due to internal tooling restrictions that prevent us from having multiple packages declared in a single repository.)
```

Added a little more context here